### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by piecioshka**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Contributing
+
+We welcome your contributions!
+
+_Each PR needs at least one approving review before merge._
```
Comment (vivensio) [inline CONTRIBUTING.md]: Just a thought: Is one approval enough?

**Review by vivensio — APPROVED** [review]
🚀

File: CONTRIBUTING.md (hunk 1 above)
Comment (piecioshka) [reply CONTRIBUTING.md]: at least one :) 

I prefer that couple devs look at my code

**Review by kgajowy — APPROVED** [review]

**Review by danekszy — APPROVED** [review]

**Comment by piecioshka** [comment]
@rsp @Tuhaj @msojda @mykulyak @bibixx @Flixow @ulaulala Please look at my PR 😈

**Review by Tuhaj — APPROVED** [review]

**Pull request merged**